package/qemu/qemu.init

```diff
@@ -106,17 +124,34 @@ start () {
     # Do we want VNC sessions encrypted?
     QemuVnc="-vnc :$QEMU_VNC_DISPLAY,websocket"
     QemuVncTls=""
-    if  [ "$QEMU_VNC_TLS" = "yes" ] && [ -n "$QEMU_VNC_TLS_DIR" ]; then
-      if [ -d "$QEMU_VNC_TLS_DIR" ] &&
-         [ -s "${QEMU_VNC_TLS_DIR}/ca-cert.pem" ] &&
-         [ -s "${QEMU_VNC_TLS_DIR}/server-cert.pem" ] &&
-         [ -s "${QEMU_VNC_TLS_DIR}/server-key.pem" ]; then
-        QemuVncTls="-object tls-creds-x509,dir=$QEMU_VNC_TLS_DIR,id=tls0,endpoint=server,verify-peer=no"
-        QemuVnc="${QemuVnc},tls-creds=tls0"
-      else
-        /usr/bin/logger -s -p user.err -id -t qemu "ERROR: TLS credentials missing in $QEMU_VNC_TLS_DIR.  Exiting"
-        exit
+    TLSmsg=""
+    if  [ "$QEMU_VNC_TLS" = "yes" ] && [ -n "${QEMU_VNC_TLS_DIR}" ]; then
+      # User requested TLS sessions.
+      if [ ! -d "${QEMU_VNC_TLS_DIR}" ] ||
+         [ ! -s "${QEMU_VNC_TLS_DIR}/ca-cert.pem" ] ||
+         [ ! -s "${QEMU_VNC_TLS_DIR}/server-cert.pem" ] ||
+         [ ! -s "${QEMU_VNC_TLS_DIR}/server-key.pem" ]; then
+        # We do not have set of certificate files.  Attempt to use ACME issued...
+        if [ -d "/etc/acme/${DOMAIN}" ] &&
+            [ -s "/etc/acme/${DOMAIN}/ca.cer" ] &&
+            [ -s "/etc/acme/${DOMAIN}/${DOMAIN}.cer" ] &&
+            [ -s "/etc/acme/${DOMAIN}/${DOMAIN}.key" ]; then
```

Comment:
Just a FYI, since a `/` and `.` are not allowed in variable names, the following two strings are identical:
```
"/etc/acme/${DOMAIN}/${DOMAIN}.key"
```
```
"/etc/acme/$DOMAIN/$DOMAIN.key"
```
No need to change anything, and sometimes using `${FOO}` is more readable.

Reply:
Thanks for the code review.  Learn something new every day.